.github/ISSUE_TEMPLATE/Documentation.yml

```diff
@@ -0,0 +1,25 @@
+name: Documentation
+description: Report an issue related to the SciPy documentation.
+title: "DOC: "
+labels: [Documentation]
+
+body:
+- type: textarea
+  attributes: 
+    label: "Issue with current documentation:"
+    description: >
+      Please make sure to leave a reference to the document/code you're
+      referring to. You can also check the development version of the
+      documentation and see if this issue has already been addressed at
+      https://https://scipy.github.io/devdocs/.
```

Comment:
What do you think of emphasizing the part about checking the development version? We do get a fair number of reports about things that have already been fixed.

> Please check the development version of the documentation at https://https://scipy.github.io/devdocs/ to see if this issue has already been addressed. If not, please link to the development version of the document/code you're referring to.

Reply:
Let me know if this is too much.